### PR TITLE
Pega Pool

### PR DESCRIPTION
**Pull request by Read1337**
added Pega Pool

### CODE REVIEW

**Comment by Read1337** [comment]
Sry about last PR, this shoud be ok

**Review by nymkappa — CHANGES_REQUESTED** [review]

File: pools.json [hunk 1]
```diff
@@ -1457,6 +1465,7 @@
         "tiger": "tiger",
         "1M1X": "1m1x",
         "OKMiner": "okminer",
-        "Titan": "titan"
+        "Titan": "titan",
+        "Pega Pool": "pegapool"
```
[the remark this answers is not in the document]
Comment (Read1337) [reply pools.json]: Changed

**Review by nymkappa — CHANGES_REQUESTED** [review]
Thanks. It's good now but can you squash your PR to only make it one commit with proper commit description please? Then we can merge it.

**Review by nymkappa — APPROVED** [review]
Tested ACK @ 572de81515f39b7ae31f86cae572d943dfc7ab6d

image

**Comment by Read1337** [comment]
Hey, Do I need to do anything else? We would love to see PEGA Pool stats on mempool.space let me know if there is anything else we need to do.

**Pull request merged**